prow/gerrit/adapter/adapter.go

```diff
@@ -73,25 +84,49 @@ var gerritMetrics = struct {
 		// Hopefully this isn't excessive enough to cause metric scraping issues.
 		"repo",
 	}),
+	triggerHelpLatency: prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Name:    "gerrit_trigger_help_latency",
+		Help:    "Histogram of seconds between triggering event (help) and ProwJob creation time.",
+		Buckets: []float64{5, 10, 20, 30, 60, 120, 180, 300, 600, 1200},
+	}, []string{
+		"org",
+	}),
+	processSingleChangeDuration: prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Name:    "gerrit_process_single_change_duration",
+		Help:    "Histogram of seconds spent processing a single gerrit change, by instance and repo.",
+		Buckets: []float64{5, 10, 20, 30, 60, 120, 180, 300, 600, 1200},
+	}, []string{
+		"org",
+		"repo",
+	}),
 	changeProcessDuration: prometheus.NewHistogramVec(prometheus.HistogramOpts{
 		Name:    "gerrit_instance_process_duration",
-		Help:    "Histogram of seconds spent processing a single gerrit instance.",
+		Help:    "Histogram of seconds spent processing changes, by instance and repo. Includes gerrit_repo_query_latency and gerrit_instance_change_sync_duration.",
 		Buckets: []float64{5, 10, 20, 30, 60, 120, 180, 300, 600, 1200, 3600},
 	}, []string{
-		"org",
+		"org", "repo",
 	}),
 	changeSyncDuration: prometheus.NewHistogramVec(prometheus.HistogramOpts{
 		Name:    "gerrit_instance_change_sync_duration",
-		Help:    "Histogram of seconds spent syncing changes from a single gerrit instance.",
+		Help:    "Histogram of seconds spent syncing changes (syncChange()) from a single gerrit instance or repo.",
 		Buckets: []float64{5, 10, 20, 30, 60, 120, 180, 300, 600, 1200, 3600},
-	}, []string{"org"}),
+	}, []string{"org", "repo"}),
+	gerritRepoQueryLatency: prometheus.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
nit: I think this is more of a duration than a latency? Latency sounds more like the network latency in this context.
Obviously not a big deal either way.

Reply:
I think I just chose "latency" because I thought we were measuring just the Gerrit API, but now I realize we also do our own rate limiting, etc on top. Because of this, I think "duration" is preferable. Will update.